src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx

```diff
@@ -117,10 +189,24 @@ class FiltersAndSampling extends AsyncView<Props, State> {
                 'Manage the inbound data you want to store. To change the sampling rate or rate limits, update your SDK configuration. The rules added below will apply on top of your SDK configuration.'
               )}
         </TextBlock>
-        <TransactionRules
+        <RulesPanel
+          rules={errorRules}
+          platformDocLink={platformDocLink}
+          onAddRule={this.handleAddErrorRule(platformDocLink)}
+          onEditRule={this.handleEditRule}
+          onDeleteRule={this.handleDeleteRule}
+        />
+        <TextBlock>
+          {t(
+            'The transaction order is limited. Traces must occur first and individual transactions must occur last. Any individual transaction rules before a trace rule will be disregarded. '
```

Comment:
Are we going to give the user feedback that their rules are invalid when they do this?

Reply:
yes, I will implement this functionality together with drag-and-drop